### PR TITLE
docs: add 'do not pull in version changes' to stable criteria

### PR DESCRIPTION
**Pull request by skoh7645**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Added 'do not pull in version changes' to stable criteria

### CODE REVIEW

File: content/docs/stacks/stacks-overview.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ These stacks are production-ready. Stable stacks must meet all the criteria set
 - Support the [appsody build](/docs/cli-commands#appsody-build) command with Buildah
 - Prevent creation of files on the local system that cannot be removed (i.e. files owned by root or other users)
 - Specify explicit versions for all required Docker images
+- Rely on base images for security fixes, and do not introduce any version changes (e.g. yum/apt/npm) in the Dockerfiles
```
Comment (neeraj-laad) [inline content/docs/stacks/stacks-overview.md]: perhaps we can say during stack build (Dockerfile-stack) or application build (Dockerfile)

File: content/docs/stacks/stacks-overview.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ These stacks are production-ready. Stable stacks must meet all the criteria set
 - Support the [appsody build](/docs/cli-commands#appsody-build) command with Buildah
 - Prevent creation of files on the local system that cannot be removed (i.e. files owned by root or other users)
 - Specify explicit versions for all required Docker images
-- Rely on base images for security fixes, and do not introduce any version changes (e.g. yum/apt/npm) in the Dockerfiles
+- Rely on base images for security fixes. Do not introduce any version changes (e.g. yum/apt/npm) in the Dockerfiles during stack build (`Dockerfile-stack`) or application build (`Dockerfile`)
```
Comment (helenmasters) [inline content/docs/stacks/stacks-overview.md]: This doesn't make sense to me... I think it'd make more sense as "Do not introduce any version changes to the ? (for example:  yum, apt or npm) in the Dockerfiles during the stack build (`Dockerfile-stack`) or application build (`Dockerfile`)".  I don't know what ? should be though.

Comment (skoh7645) [reply content/docs/stacks/stacks-overview.md]: "?" could be "dependencies required by the stack"?

Comment (neeraj-laad) [reply content/docs/stacks/stacks-overview.md]: We are doing this to be more compliant with RH image certification and Docker best practices. Perhaps we can use some words from their docs:

RH: 
"No modifications should be made to content provided by Red Hat packages or layers, except for files that are meant to be modified by end-users, such as configuration files." 

Docker: 
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Comment (neeraj-laad) [reply content/docs/stacks/stacks-overview.md]: Do not introduce any version changes to the content provided by the parent container images (for example: yum upgrade, apt-get dist-upgrade, npm audit fix etc.)

If a package contained in the parent image is out-of-date, contact its maintainers or update it individually.

Comment (helenmasters) [reply content/docs/stacks/stacks-overview.md]: Thanks Neeraj, it was completely unclear to me why this was being added. Either of those options are clearer than the previous suggestions.

File: content/docs/stacks/stacks-overview.md [hunk 3]
```diff
@@ -74,6 +74,7 @@ These stacks are production-ready. Stable stacks must meet all the criteria set
 - Support the [appsody build](/docs/cli-commands#appsody-build) command with Buildah
 - Prevent creation of files on the local system that cannot be removed (i.e. files owned by root or other users)
 - Specify explicit versions for all required Docker images
+- Rely on base images for security fixes. Do not introduce any version changes to the content provided by the parent container images (for example: yum upgrade, apt-get dist-upgrade, npm audit fix). If a package contained in the parent image is out-of-date, contact its' maintainers or update it individually.
```
Comment (neeraj-laad) [inline content/docs/stacks/stacks-overview.md]: Can we remove the first line Rely on base images for security fixes? This is covered in the second line "if a package contained in the parent image is out of date..."

File: content/docs/stacks/stacks-overview.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ These stacks are production-ready. Stable stacks must meet all the criteria set
 - Support the [appsody build](/docs/cli-commands#appsody-build) command with Buildah
 - Prevent creation of files on the local system that cannot be removed (i.e. files owned by root or other users)
 - Specify explicit versions for all required Docker images
-- Rely on base images for security fixes. Do not introduce any version changes to the content provided by the parent container images (for example: yum upgrade, apt-get dist-upgrade, npm audit fix). If a package contained in the parent image is out-of-date, contact its' maintainers or update it individually.
```
Comment (helenmasters) [inline content/docs/stacks/stacks-overview.md]: - `yum upgrade`, `apt-get dist-upgrade` and `npm audit fix` are all commands, shouldn't they therefore all be back-ticked?
- its' -> its
- out-of-date shouldn't have hyphens unless it is placed before a noun, so please remove the hyphens

**Pull request merged**